modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path

```diff
@@ -95,6 +95,11 @@
 	<td>//a[contains(@class, 'font-weight-bold')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEARCH_FILTER</td>
+	<td>//div[contains(@class,'bg-neutral-1')]//div[@class="input-group-item"]//input[@placeholder="Search"]</td>
+	<td></td>
```

Comment:
without the "/div[contains(@class,'bg-neutral-1')]", does the locator work?

Reply:
No, without that, the path finds more than 1 result